Changelog.md

```diff
@@ -17,6 +17,7 @@ Compiler Features:
 
 
 Bugfixes:
+ * Code Generator: Fix not entirely deterministic order of functions in unoptimized Yul output. The choice of C++ compiler in some cases would result in different (but equivalent) bytecode (especially from native binaries vs emscripten binaries)
```

Comment:
```suggestion
 * Code Generator: Fix not entirely deterministic order of functions in unoptimized Yul output. The choice of C++ compiler in some cases would result in different (but equivalent) bytecode (especially from native binaries vs emscripten binaries).
```

Reply:
Well, we can still fix this when we sort the changelog before release. If this was a bigger mistake I'd push a new PR, but for a dot it's not worth it :)